### PR TITLE
subtle changes: diction & concept assimilation

### PR DESCRIPTION
**Pull request by GoesToEleven**
1. The diction change offers more precision with vocabulary.
2. The change for assimilation is subjective.

### CODE REVIEW

File: CONTRIBUTORS [hunk 1]
```diff
@@ -23,7 +23,8 @@
 Alex Bozhenko <alexbozhenko@gmail.com>
 César Cará <cesar.cara@protonmail.com>
 Cheikh Seck <cheikh.seck@ardanlabs.com>
+Gustavo Castillo <gcdcoder@gmail.com>
 Hana Mohan <hana@magicbell.io>
 Sergey Kibish <5479211+skibish@users.noreply.github.com>
-Gustavo Castillo <gcdcoder@gmail.com>
+Todd McLeod <8296040+GoesToEleven@users.noreply.github.com>
```
Comment (ardan-bkennedy) [inline CONTRIBUTORS:29]: Todd do you see the comment?

Comment (GoesToEleven) [reply CONTRIBUTORS:29]: Yes!

File: _content/tour/pointers/example5.go [hunk 2]
```diff
@@ -14,20 +14,20 @@ const size = 1
...
-	stackCopy(&s, 0, [size]int{})
+	stackCopy(0, &s, [size]int{})
 }
 
 // stackCopy recursively runs increasing the size
 // of the stack.
 //
 //go:noinline
-func stackCopy(s *string, c int, a [size]int) {
+func stackCopy(c int, s *string, a [size]int) {
```
[the remark this answers is not in the document]
Comment (GoesToEleven) [reply _content/tour/pointers/example5.go:24]: This is minor, but here is how I saw this.

The func is like this

```go
func stackCopy(s *string, c int, a [size]int) {
	println(c, s, *s)
```

The parameters:
s, c, a

The println arguments:
c, s, *s

BETTER READABILITY (mental symmetry) would be:
The parameters:
c, s, a

The println arguments:
c, s, *s

[the remark this answers is not in the document]
Comment (ardan-bkennedy) [reply _content/tour/pointers/example5.go:24]: The string is the primary think for this function, the c argument is just a count for the number of iterations. I think you are giving the count more precedence by putting it first. But like I said, I can't make this change.

File: _content/tour/variables.article [hunk 3]
```diff
@@ -6,10 +6,7 @@ Variables are at the heart of the language and provide the ability to read from
 - [[https://www.ardanlabs.com/training/individual-on-demand/ultimate-go-bundle/][Watch The Video]]
 - Need Financial Assistance, Use Our [[https://www.ardanlabs.com/scholarship/][Scholarship Form]]
 
-Variables are at the heart of the language and provide the ability to read from
-and write to memory. In Go, access to memory is type safe. This means the compiler
-takes type seriously and will not allow us to use variables outside the scope of
-how they are declared.
+Variables are at the heart of the language and provide the ability to read from and write to memory. In Go, access to memory is type safe. This means the compiler takes type seriously and will not allow us to use variables outside the scope of how they are declared.
```
[the remark this answers is not in the document]
Comment (GoesToEleven) [reply _content/tour/variables.article:9]: The line breaks show up differently on different screen sizes, so on some screen sizes the line breaks look broken. 

See image attached - look for the "second word is 0" part.

Screenshot 2023-09-13 at 11 38 20 AM

**Pull request closed without merging**